pages/apis/rest_api/builds.md.erb

```diff
@@ -685,11 +685,11 @@ Each <em>build</em> is provided with the following timestamps:
 <tbody>
   <tr>
     <th><code>scheduled_at</code></th>
-    <td>The time the build was created</td>
+    <td>The time the build was created. When you do a <code>pipeline upload</code>, <code>scheduled_at</code> is copied from the job that did the uploading</td>
   </tr>
   <tr>
     <th><code>created_at</code></th>
-    <td>The time the build was created</td>
+    <td>The time the build was created. Coincides with <code>pipeline upload</code></td>
```

Comment:
But this isn't super clear to me. Are we saying that created at is the same as scheduled at? 
Or timestamp from when pipeline upload was run?

Reply:
@karensawrey I would not use 'coincides' here. Maybe something as simple as "is the same as"?